Node/src/TeamsChatConnector.ts

```diff
@@ -146,10 +146,77 @@ export class TeamsChatConnector extends builder.ChatConnector {
     });
   }
 
+  /**
+  *  Return members in a team or channel with pagination
+  *  @param {string} serverUrl - Server url is composed of baseUrl and cloud name, remember to find your correct cloud name in session or the function will not find the team.
+  *  @param {string} conversationId - The conversation id or channel id, you can look it up in session object.
+  *  @param {number} pageSize - optional. If exists, it specifies how many members to fetch per page, if it's not passed, default is 200
+  *  @param {string} continuationToken - optional. If exists, callers can make a subsequent call to fetch more members
+  *  @param {function} callback - This callback returns err or result.
+  */
+  public fetchMembersWithPaging(
+    serverUrl: string,
+    conversationId: string,
+    ...args: any[]): void {
+    let pageSize: number = undefined;
+    let continuationToken: string = undefined;
+    let callback: (err: Error, result: TeamsChannelAccountsResult) => void;
+
+    let throwInvalidParameter = () => {
+      throw new Error('Invalid parameters: ' + JSON.stringify(args));
+    };
+
+    if (!args) {
+      throwInvalidParameter();
+    }
+
+    switch (args.length) {
+      case 1:
```

Comment:
Let's label the case with a comment explaining the expected call pattern, e.g.:
// fetchMembersWithPaging(serverUrl, conversationId, callback)
case 1:
// fetchMembersWithPaging(serverUrl, conversationId, pageSize, callback)
// fetchMembersWithPaging(serverUrl, conversationId, continuationToken, callback)
case 2: